CODING_GUIDELINES.md

```diff
@@ -207,7 +207,36 @@ class MyClass
 }
 ```
 
-### 8. Misc.###
+### 8. Header inclusion order.###
+The headers should be placed in the following order:
+ 1. Module header (in .cpp)
+ 2. System/Qt/Boost etc. headers (splitted in subcategories if you have many).
```

Comment:
The point is pretty clear: one can easily see the list of Qt modules this file depends on.